index.html

```diff
@@ -246,15 +262,37 @@ <h2>Get started</h2>
                 <p id="form-email" class="form-details">Your email</p>
                 <p id="form-text" class="form-details text-area-text">Enter your message here...</p>
             </div>
-            <form action="https://formspree.io/f/xzboarvz" method="post">
+            <form data-form id="contact-form" action="https://formspree.io/f/xzboarvz" method="post">
                 <ul>
-                    <li><input type="text" required maxlength="30" id="list-name"></li>
+                    <li><input type="text" required minlength="3" maxlength="30" id="list-name"></li>
                     <li><input type="email" required maxlength="50" id="list-email"></li>
                     <li><textarea name="textarea" cols="30" rows="10" required maxlength="500" id="list-textarea"></textarea></li>
                     <li><button type="submit" class="btn-get-started">Start collaboration</button></li>
                 </ul>
             </form>
         </div>
+        <div class="modal" id="modal">
+            <div class="modal-header">
+                <div data-title class="title"></div>
+                <button data-close-button class="btn-close" type="button">&times;</button>
+            </div>
+            <div class="modal-body">
+                <img src="#" alt="Project image" class="rectangle-34" id="image-1">
+                <img src="#" alt="Project image" class="rectangle-39" id="image-2">
+                <img src="#" alt="Project image" class="rectangle-40" id="image-3">
+                <img src="#" alt="Project image" class="rectangle-41" id="image-4">
+                <img src="#" alt="Project image" class="rectangle-42" id="image-5">
+                <div data-html-css class="html-css html-border"></div>
+                <div data-ruby-rails class="ruby-rails ruby-border"></div>
+                <div data-javascript class="javascript javascript-border"></div>
+                <article data-description></article>
+                <button data-see-live class="btn-see-live" type="button">See live <i class="fa fa-external-link"></i></button>
+                <button data-see-source class="btn-see-source" type="button">See source <i class="fa fa-github"></i></button>
+                <a href="" id="previous"><i class="fa-solid fa-arrow-left-long"></i>Previous project</a>
+                <a href="" id="next">Next project<i class="fa-solid fa-arrow-right-long"></i></a>
+            </div>
+        </div>
+        <div id="overlay"></div>
     </main>
     <footer>
         <h3>Get my Resume</h3>
```

Comment:
- [ ] Kindly fix the white space under the footer and remove the commented code inside the footer. chek the illustration bellow 
<img width="1440" alt="Screen Shot 2022-04-15 at 11 46 24" src="https://user-images.githubusercontent.com/28307960/163556724-99862f07-76fe-4eeb-8523-089c6f9e4565.png">

Reply:
Removed the whitespace and commented code. Thanks!